cmd/faucet/faucet.go

```diff
@@ -471,7 +471,7 @@ func (f *faucet) apiHandler(conn *websocket.Conn) {
 
 		// Send an error if too frequent funding, othewise a success
 		if !fund {
-			websocket.JSON.Send(conn, map[string]string{"error": fmt.Sprintf("%s left until next allowance", common.PrettyDuration(timeout.Sub(time.Now())))})
+			websocket.JSON.Send(conn, map[string]string{"error": fmt.Sprintf("%s left until next allowance", common.PrettyDuration(time.Until(timeout)))})
```

Comment:
This fix requires Go 1.8. We do not yet want to drop builds with Go 1.7.x, so please revert this change.